kobo_selenium_tests/edited_test_cases/enter_data_test_edited.py

```diff
@@ -117,6 +117,17 @@ def test_enter_data_test_edited(self):
         self.assertEqual("Submissions (1)", driver.find_element_by_css_selector(
             ".dashboard__submissions .dashboard__group-label").text)
 
+        # Verify that the QR code is present.
+        driver.find_element_by_css_selector(".dashboard__button-how-to-collect").click()
+        for i in range(60):
+            try:
+                if "How to Collect Data on a Mobile Device" == driver.find_element_by_css_selector("div.vex-content.vex-content-large > #popupmodal > div.vex-header > h3").text: break
```

Comment:
Shall we put a more descriptive class on this text (or the parent modal) so I can do this in a less language-dependent way?